### PR TITLE
Add camel-jta wrapper

### PR DESCRIPTION
**Pull request by f2par0**
camel-jta wrapper

### CODE REVIEW

**Review by lopushen — COMMENTED** [review]
Review in progress

File: features/src/main/feature/camel-features.xml [hunk 1]
```diff
@@ -1125,6 +1125,16 @@
         <bundle dependency='true'>mvn:org.ow2.asm/asm/9.3</bundle>
         <bundle>mvn:org.apache.camel.karaf/camel-jsonpath/${project.version}</bundle>
     </feature>
+    <feature name='camel-jta' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <bundle dependency='true'>mvn:jakarta.transaction/jakarta.transaction-api/${jakarta-transaction-api-version}</bundle>
+        <bundle dependency='true'>mvn:jakarta.enterprise/jakarta.enterprise.cdi-api/${jakarta-enterprise-cdi-api-version}</bundle>
+        <bundle dependency='true'>mvn:jakarta.enterprise/jakarta.enterprise.lang-model/4.0.1</bundle>
```
Comment (lopushen) [inline features/src/main/feature/camel-features.xml:1132]: the value is the same as `jakarta-enterprise-cdi-api-version` parameter, but naming suggests that a separate parameter should be used

Comment (lopushen) [reply features/src/main/feature/camel-features.xml:1132]: there are newer versions, but they are either milestones or release candidates

File: features/src/main/feature/camel-features.xml [hunk 2]
```diff
@@ -1125,6 +1125,16 @@
...
+        <bundle dependency='true'>mvn:jakarta.inject/jakarta.inject-api/${jakarta-inject-version}</bundle>
+        <bundle dependency='true'>mvn:jakarta.el/jakarta.el-api/${jakarta-el-api-version}</bundle>
+        <bundle dependency='true'>mvn:jakarta.interceptor/jakarta.interceptor-api/2.1.0</bundle>
```
Comment (lopushen) [inline features/src/main/feature/camel-features.xml:1135]: same here, there are newer versions, but they are either milestones or release candidates

**Review by lopushen — APPROVED** [review]
no conflict ✔️
compilation works ✔️
no unnecessary usage of wrap protocol ✔️
no unnecessary dependencies ✔️
no hardcoded version if it's in the parent pom ✔️
feature is placed in alphabetical order in the file camel-features.xml ✔️
usage of dependent features (netty, jackson, jakarta ...) ✔️

**Review by essobedo — APPROVED** [review]
LGTM

**Pull request merged**